pkg/hintrunner/zero/zerohint_others.go

```diff
@@ -37,12 +37,11 @@ func newMemContinueHint(continueTarget hinter.ResOperander, memset bool) hinter.
 				return err
 			}
 
-			newN, ok := n.(fp.Element)
+			newN, ok := n.(uint64)
 			if !ok {
 				return fmt.Errorf("casting n into a felt failed")
```

Comment:
Should we update the error message as well?

Reply:
sure! will do it now